incubator/webpagetest-server/Chart.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+description: A Helm chart for Kubernetes
+name: webpagetest-server
+version: 0.1.0
```

Comment:
Please fill in the maintainers of the chart. Their names should be the GitHub usernames.